contracts/contracts/URaidenEIP712HelperContract.sol

```diff
@@ -0,0 +1,34 @@
+pragma solidity ^0.4.17;
+
+contract URaidenEIP712HelperContract {
+    /// @dev Returns the message hash used for creating the balance proof.
+    /// Used in the RaidenMicroTransferChannels contract.
+    /// Should be kept up-to-date with eth_signTypedData https://github.com/ethereum/EIPs/pull/712.
+    /// @param _receiver_address The address that receives tokens.
+    /// @param _open_block_number The block number at which a channel between the
+    /// sender and receiver was created.
+    /// @param _balance The amount of tokens owed by the sender to the receiver.
+    /// @param _contract RaidenMicroTransferChannels contract address
+    /// @return Hash of the message composed from the above parameters.
+    function getMessageHash(
+        address _receiver_address,
+        uint32 _open_block_number,
+        uint192 _balance,
+        address _contract)
+        public
+        pure
+        returns (bytes32)
+    {
+        // Used in the RaidenMicroTransferChannels contract
+        // The variable names from below will be shown to the sender when signing
+        // the balance proof, so they have to be kept in sync with the Dapp client.
+        // The hashed strings should be kept in sync with this function's parameters
+        // (variable names and types)
+        bytes32 message_hash = keccak256(
+          keccak256('address receiver', 'uint32 block_created', 'uint192 balance', 'address contract'),
```

Comment:
The test [here](https://github.com/raiden-network/microraiden/pull/200/files#diff-00b1cc92080ad0cb14ac960cb278f551R155) guarantees that they are kept in sync, right? If not then we need such a test.

Reply:
Tests have been added here https://github.com/raiden-network/microraiden/pull/200/files#diff-bd130ba0c786d648c876d74356582f11R50